docs/pages/database-access/guides/postgres-cloudsql.mdx

```diff
@@ -308,6 +310,12 @@ earlier.
 See [Authenticating as a service account](https://cloud.google.com/docs/authentication/production)
 in the Google Cloud documentation for more details.
 
+If you are using `systemd` to start `teleport`, then you should edit `teleport.service`
+(`/etc/systemd/system/teleport.service`) and add the following into the `[Service]` section:
```

Comment:
If you're using one of our package installs, won't modifications to teleport.service just get overwritten on upgrade?

Reply:
Yeah, you are right, they will.

We should be able to add the env var if we create a file in a specific location:
`/etc/systemd/system/teleport.service.d/local.conf`
```
[Service]
Environment=GOOGLE_APPLICATION_CREDENTIALS=/path/to/credentials.json
```

_doing some tests_

Reply:
The service already has an `EnvironmentFile` defined (`/lib/systemd/system/teleport.service`)
```conf
[Unit]
Description=Teleport Service
After=network.target

[Service]
Type=simple
Restart=on-failure
EnvironmentFile=-/etc/default/teleport
ExecStart=/usr/local/bin/teleport start --pid-file=/run/teleport.pid
ExecReload=/bin/kill -HUP $MAINPID
PIDFile=/run/teleport.pid
LimitNOFILE=524288

[Install]
WantedBy=multi-user.target
```

So, instead of changing the `teleport.service` or creating another file using the override method, let's suggest users to use that `EnvironmentFile` to define the variable:

```
$ echo 'GOOGLE_APPLICATION_CREDENTIALS=/path/to/credentials.json' | sudo tee -a /etc/default/teleport
```